### DISCUSSION TITLE
Can birefnet for background removal batch process images?

### DISCUSSION THREAD

**Discussion by aidec** [lllyasviel/stable-diffusion-webui-forge#1290] [Q&A]
Input multiple local pictures and perform batch processing to memorize them.

**Comment by lllyasviel** [comment]
There is an API button on that space

![image](https://github.com/user-attachments/assets/ba69dccc-f061-41fd-97f6-649b645b6ac2)

I have absolutely no idea how that works but many people report successful and good experience with that in recent gradio

**Reply by aidec** [reply]
Thanks, I will try it.
